stable/goldilocks/ci/ingress-values.yaml

```diff
@@ -0,0 +1,23 @@
+# For testing, make sure to install the vpa and metrics-server as sub-charts
```

Comment:
It doesn't seem necessary to add a new test file here. Please just modify the old one. What is added here doesn't conflict with that